CHANGELOG.md

```diff
@@ -63,6 +63,8 @@
 ### Mimirtool
 
 * [ENHANCEMENT] Added `mimirtool rules delete-namespace` command to delete all of the rule groups in a namespace including the namespace itself. #3136
+* [ENHANCEMENT] Refactor `mimirtool analyze prometheus`: add concurrency and resiliency #3062
```

Comment:
We're used to reference the PR number instead of the issue number (so `#3349` instead of `#3062`). This will make our release tooling matching the PR when checking the CHANGELOG.

@anilmisirlioglu would you mind opening a follow up PR to change it, please?

Reply:
Sure